sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-CSPT.e2e-spec.ts

```diff
@@ -0,0 +1,56 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../test-utils";
+import { YesNoOptions } from "@sims/test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-CSPT.`, () => {
+  it(
+    "Should determine CSPT as eligible when total assessed need is greater than 0 " +
+      "and total family income is less than the threshold for a married student with a canadian spouse.",
```

Comment:
Does "married student with a canadian spouse." impact the E2E scenario or is it related to the family size?

Reply:
https://github.com/bcgov/SIMS/pull/2506#discussion_r1394937155